### DISCUSSION TITLE
Αίθουσα τηλεδιάσκεψης για συμπληρωματική ΟΣΣ1

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#8] [Show and tell]
Σύμφωνα με το αποτέλεσμα της ψηφοφορίας #6  θα γίνει συμπληρωματική ΟΣΣ1 το (μεθ)επόμενο 

Σάββατο 15 4μμ 

Σάββατο 22 Οκτ 3μμ 

στην αίθουσα τηλεδιάσκεψης που λέει στο study.eap.gr:
https://hou.webex.com/meet/chorianopoulos.konstantinos

στην περίπτωση που υπάρχουν τεχνικά προβλήματα στο παραπάνω, τότε

στην παρακάτω αίθουσα ζουμ (σύνδεση μόνο από Ελλάδα και μόνο με λογαριασμό στην πλατφόρμα ζουμ):

https://ionio-gr.zoom.us/j/96573757306

ο κωδικός σύνδεσης είναι στην κεντρική σελίδα

**Comment by dimidimots** [comment]
To passcode ποιο είναι?
